test/runtests.jl

```diff
@@ -28,6 +28,20 @@ function runtests()
     return
 end
 
+function test_write_bool_model()
```

Comment:
@odow Is this OK? I'll add some more tests if so

Reply:
specifically `Model{Bool}`

Reply:
This point came up the other day: `Model{Bool}` means that the coefficients are `Bool`, but I never really know whether we intended it for the variable primals to also be `Bool`. I guess so, but it means that we probably intend for the type `Model{T}` to be such that `T * T = T`.

Reply:
The string `Model{Bool}` does not seem to occur anywhere in MOI, so I suppose it is untested. It seems to work OK here though. I think it makes sense for binary problems and SAT problems in particular. We ought to decide if it's OK and add tests to MOI if so.

Reply:
We have tests for a range of coefficient types so I'm not too worried. The issue is that it restricts things so that you couldn't have, for example, `true * x + true * y + true * z >= 2`. I think in most cases you really want `Model{Int}` and to interpret `0` as `false` and `> 0` as `true`.

Reply:
sure but you can't really do +/- in SAT anyway - you use boolean operators. so I want to check we can do SAT modeling in a nice/convenient way